src/huggingface_hub/inference/_client.py

```diff
@@ -1990,3 +1991,50 @@ def get_model_status(self, model: Optional[str] = None) -> ModelStatus:
             compute_type=response_data["compute_type"],
             framework=response_data["framework"],
         )
+
+    def audio_to_audio(
```

Comment:
Would it be possible to move the method definition up in the file between `audio_classification` and `automatic_speech_recognition`? I'm trying to order them by alphabetical order (-ish...). Thanks in advance!